styles.css

```diff
@@ -482,3 +492,285 @@ textarea {
+    padding: 5px;
+    color: #ff6b00;
+    border: 1px solid #f55800;
+  }
+
+  .presentation {
+    background-image: none;
+    display: grid;
+    grid-template-columns: 0.5fr 2.25fr 2.25fr;
+    align-items: center;
+  }
+
+  .background-desktop img {
+    float: right;
+    width: 90%;
+  }
+
+  .social-media {
+    flex-direction: column;
+    row-gap: 20px;
+    align-items: center;
+  }
+
+  .works-section {
+    display: grid;
+    grid-template-columns: 1fr 1.2fr;
+    align-items: center;
+  }
+
+  .vector {
+    height: 1px;
+    background-color: #212121;
+    margin-right: 10%;
+    margin-left: 0;
+  }
+
+  .title-works {
+    padding: 135px 0;
+  }
+
+  .stories-section {
+    display: grid;
+    grid-template-columns: 1.5fr 1fr;
+    column-gap: 24px;
+  }
+
+  .stories-section img {
+    width: 100%;
+    margin: 0;
+  }
+
+  .stories-section section {
+    margin-right: 10%;
+  }
+
+  .projects {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 37px 24px;
+  }
+
+  .languages ul {
+    padding: 0 25%;
+    align-items: flex-start;
+  }
+
+  .projects-container h2,
+  .projects-container p {
+    margin: 16px 16px;
+  }
+
+  .projects-container:first-child {
+    background-image: url("img/imgplaceholder1.png");
+    background-size: cover;
+    padding: 0;
+    display: flex;
+    flex-direction: column-reverse;
+    border: solid 1px black;
+  }
+
+  .projects-container:first-child h2,
+  .projects-container:first-child p,
+  .projects-container:first-child ul {
+    display: none;
+  }
+
+  .projects-container:nth-child(2n) {
+    background-image: url("img/imgplaceholder2.png");
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .projects-container:nth-child(3n) {
+    background-image: url("img/imgplaceholder3.png");
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .projects-container:nth-child(4n) {
+    background-image: url("img/imgplaceholder4.png");
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .projects-container:nth-child(5n) {
+    background-image: url("img/imgplaceholder2.png");
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .projects-container:last-child {
+    background-image: url("img/imgplaceholder3.png");
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .projects-container:nth-child(2n) a,
+  .projects-container:nth-child(3n) a,
+  .projects-container:nth-child(4n) a,
+  .projects-container:nth-child(5n) a,
+  .projects-container:last-child a {
+    display: none;
+  }
+
+  .container {
+    max-width: 85%;
+    margin: 20px auto;
+  }
+
+  .techs-container {
+    margin: 20px 0 32px 0;
+  }
```

Comment:
- [ ] To match the original design better, you should consider using another `justify-content`, both for mobile and desktop versions. Right now they are centered, but should all be aligned to the left.
- [ ] Also, make sure that you are using the same typography provided in the Figma design, your `<h2 class="multi">` and `<p class="story">` is a bit smaller than it should be, as you can see here:

Figma design:
![image](https://user-images.githubusercontent.com/66967274/144540420-10991b8a-b615-4dd9-8e3d-2b57c1d39cc6.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/144540448-73afca66-14ad-48cd-84a7-bf4a4e475d1d.png)